From 981d26e3a97488ec9fdfc1aad25ca33a08766265 Mon Sep 17 00:00:00 2001
From: Stephen Gallagher <sgallagh@redhat.com>
Date: Fri, 28 Feb 2014 16:20:03 -0500
Subject: [PATCH 2/2] Soften dependencies

---
 reviewboard/__init__.py | 2 +-
 setup.py                | 6 +++---
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/reviewboard/__init__.py b/reviewboard/__init__.py
index bd22c72fd223f7de84bbe25e431239551d6701b9..57b2f20dd29d8c6231682246379cd442d250de7d 100644
--- a/reviewboard/__init__.py
+++ b/reviewboard/__init__.py
@@ -9,11 +9,11 @@ from __future__ import unicode_literals
 #
 VERSION = (2, 0, 11, 0, 'final', 0, True)
 
 
 # Required version of Django
-django_version = 'Django>=1.6.7,<1.7'
+django_version = 'Django>=1.6,<1.7'
 
 
 def get_version_string():
     version = '%s.%s' % (VERSION[0], VERSION[1])
 
diff --git a/setup.py b/setup.py
index 6f20ec482c770323d94375c5a6c7cac0c4019bd5..659ee8e978739036d1220520beea00ea0706a851 100755
--- a/setup.py
+++ b/setup.py
@@ -196,18 +196,18 @@ setup(name=PACKAGE_NAME,
           'django_evolution>=0.7.4,<=0.7.999',
           'Djblets>=0.8.12,<=0.8.999',
           'django-haystack>=2.1',
           'docutils',
           'markdown>=2.4.0,<2.4.999',
-          'mimeparse>=0.1.3',
+          'python_mimeparse>=0.1.3',
           'paramiko>=1.12',
           'Pygments>=1.6',
-          'python-dateutil==1.5',
+          'python-dateutil<2.0',
           'python-memcached',
           'pytz',
           'recaptcha-client',
-          'Whoosh>=2.6',
+          'Whoosh',
       ],
       dependency_links = [
           'http://downloads.reviewboard.org/mirror/',
           'http://downloads.reviewboard.org/releases/Djblets/0.8/',
           'http://downloads.reviewboard.org/releases/django-evolution/0.7/',
-- 
2.1.0

