From dad6635f12c0fff9f41b93a029ca0453a4bc9ff0 Mon Sep 17 00:00:00 2001
From: Stephen Gallagher <sgallagh@redhat.com>
Date: Fri, 28 Feb 2014 16:20:03 -0500
Subject: [PATCH 1/2] Soften dependencies

---
 reviewboard/__init__.py | 2 +-
 setup.py                | 6 +++---
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/reviewboard/__init__.py b/reviewboard/__init__.py
index a0e4f4b89221c8187684214c2a064f4fb47b5055..417d9a619cd0f171341213bd9ff9ee495066cacb 100644
--- a/reviewboard/__init__.py
+++ b/reviewboard/__init__.py
@@ -9,11 +9,11 @@ from __future__ import unicode_literals
 #
 VERSION = (2, 0, 19, 0, 'final', 0, True)
 
 
 # Required version of Django
-django_version = 'Django>=1.6.11,<1.7'
+django_version = 'Django>=1.6,<1.7'
 
 
 def get_version_string():
     version = '%s.%s' % (VERSION[0], VERSION[1])
 
diff --git a/setup.py b/setup.py
index 8166b49b15ec1f57aeb5f334db005beb1b1dc677..a80937cbbec9f0cb5b388173f8f0c19de16a973e 100755
--- a/setup.py
+++ b/setup.py
@@ -199,19 +199,19 @@ setup(name=PACKAGE_NAME,
           'django_evolution>=0.7.5,<=0.7.999',
           'django-haystack>=2.3.1',
           'Djblets>=0.8.21,<=0.8.999',
           'docutils',
           'markdown>=2.4.0,<2.4.999',
-          'mimeparse>=0.1.3',
+          'python_mimeparse>=0.1.3',
           'paramiko>=1.12',
           'pycrypto>=2.6',
           'Pygments>=1.6',
-          'python-dateutil==1.5',
+          'python-dateutil<2.0',
           'python-memcached',
           'pytz',
           'recaptcha-client',
-          'Whoosh>=2.6',
+          'Whoosh',
       ],
       dependency_links = [
           'http://downloads.reviewboard.org/mirror/',
           'http://downloads.reviewboard.org/releases/Djblets/0.8/',
           'http://downloads.reviewboard.org/releases/django-evolution/0.7/',
-- 
2.5.0

