From 94e489d3e2b3b1bc0357f02683d55705bd8acacd Mon Sep 17 00:00:00 2001
From: Stephen Gallagher <sgallagh@redhat.com>
Date: Fri, 28 Feb 2014 16:20:03 -0500
Subject: [PATCH] Soften dependencies

---
 setup.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/setup.py b/setup.py
index e04b871441561cd2b369a43ae391d4c2c907e7fd..5eb5311dfebb5cbd1780f235e66d89d8776bdcbe 100755
--- a/setup.py
+++ b/setup.py
@@ -192,17 +192,17 @@ setup(name=PACKAGE_NAME,
       },
       cmdclass=cmdclasses,
       install_requires=[
-          'Django>=1.6.2,<1.7',
+          'Django>=1.6,<1.7',
           'django_evolution>=0.7,<0.8',
           'Djblets>=0.8beta3,<0.9',
           'django-haystack',
           'django-pipeline>=1.3.15',
           'docutils',
           'markdown>=2.3.1',
-          'mimeparse>=0.1.3',
+          'python_mimeparse>=0.1.3',
           'paramiko>=1.12',
           'Pygments>=1.6',
-          'python-dateutil==1.5',
+          'python-dateutil<2.0',
           'python-memcached',
           'pytz',
           'recaptcha-client',
-- 
1.8.5.3

