From 5839896c0fa2f2ed98c3ad36ccd71b193c26743f Mon Sep 17 00:00:00 2001
From: Stephen Gallagher <sgallagh@redhat.com>
Date: Fri, 28 Feb 2014 16:20:03 -0500
Subject: [PATCH 2/2] Soften dependencies

---
 reviewboard/__init__.py | 3 +--
 setup.py                | 4 ++--
 2 files changed, 3 insertions(+), 4 deletions(-)

diff --git a/reviewboard/__init__.py b/reviewboard/__init__.py
index c4bc22b2ebeed6971b282d3363659a5141fabc91..32677bf012f19559c764c515d0672a7207bfd6f5 100644
--- a/reviewboard/__init__.py
+++ b/reviewboard/__init__.py
@@ -11,8 +11,7 @@ VERSION = (2, 0, 2, 0, 'final', 0, False)
 
 
 # Required version of Django
-django_version = 'Django>=1.6.5,<1.7'
-
+django_version = 'Django>=1.6,<1.7',
 
 def get_version_string():
     version = '%s.%s' % (VERSION[0], VERSION[1])
diff --git a/setup.py b/setup.py
index 508739158b985725eea2fb507a92c57bd9210901..c13a3ea84205b2873c6acba08b78c685026b89ff 100755
--- a/setup.py
+++ b/setup.py
@@ -198,10 +198,10 @@ setup(name=PACKAGE_NAME,
           'django-haystack',
           'docutils',
           'markdown>=2.3.1',
-          'mimeparse>=0.1.3',
+          'python_mimeparse>=0.1.3',
           'paramiko>=1.12',
           'Pygments>=1.6',
-          'python-dateutil==1.5',
+          'python-dateutil<2.0',
           'python-memcached',
           'pytz',
           'recaptcha-client',
-- 
1.9.0

